docs/developer/running-saleor/task-queue.mdx

```diff
@@ -42,7 +42,7 @@ Below is a list of Saleor features fired up by celery beat with default schedule
 - [Recalculate price for catalog promotions](developer/discounts/promotions.mdx#catalogue-promotions); 30 seconds
 - [Release funds for abandoned checkouts](developer/checkout/lifecycle.mdx#releasing-funds-for-abandoned-checkouts); 10 minutes
 - Remove apps marked as removed; 3 hours
-- [Update search vectors](developer/community/contributing.mdx#searching); 20 seconds
+- [Update search vectors](https://github.com/saleor/saleor/blob/main/CONTRIBUTING.md#searching); 20 seconds
```

Comment:
It's weird that the info about searching vector is in contribution. I think it should be a part of a standard docs. Could you add a ticket to move it somewhere in the docs?

Reply:
I will move it straight away in the next PR 😄